src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__link"><a class="nav__link is-active" href="#Apple">Apple</a></li>
+          <li class="nav__link"><a class="nav__link" href="#Samsung">Samsung</a></li>
+          <li class="nav__link"><a class="nav__link" href="#Смартфоны и телефоны">Смартфоны и телефоны</a></li>
```

Comment:
Don't use cyrrilic symbols in links, that's a bad practice, fix it everywhere
```suggestion
          <li class="nav__link"><a class="nav__link" href="#phones">Смартфоны и телефоны</a></li>
```